apps/api/v2/src/modules/router/controllers/router.controller.ts

```diff
@@ -37,6 +37,13 @@ export class RouterController {
     }
 
     if (routedUrlData?.props) {
+      if (routedUrlData.props.errorMessage) {
+        return {
+          status: "error",
+          error: { code: "ROUTING_ERROR", message: routedUrlData.props.errorMessage },
+          redirect: false,
+        };
```

Comment:
This fixes an existing case where in case of form field validation failure, we were still returning success with empty message.